### PR TITLE
AB-7445 Add latitude, longitude and Autobutler profile url to settings

### PR DESCRIPTION
**Pull request by andreaseriksson**
https://github.com/autobutler/autobutler/pull/6158

### CODE REVIEW

File: source/includes/_settings.md [hunk 1]
```diff
@@ -207,6 +207,8 @@ jQuery.ajax({
       "name": "BMW"
     }
   ],
+  "latitude": 0.0,
```
Comment (dipth) [inline source/includes/_settings.md]: Please move these attributes above `"makes": [` to keep the list sorted alphabetically

File: source/includes/_settings.md [hunk 2]
```diff
@@ -354,6 +356,8 @@ invoiceReference                             | string        | yes           | A
 makes                                        | array(object) | no            | A list of objects representing the different makes that the workshop is officially authorized to service
 makes.id                                     | integer       | no            | The id of the make
 makes.name                                   | string        | no            | The name of the make
+latitude                                     | float         | no            | Latitude of the workshop
```
Comment (dipth) [inline source/includes/_settings.md]: Please move these attributes above `makes` to keep the list sorted alphabetically

File: source/includes/_settings.md [hunk 3]
```diff
@@ -583,6 +587,8 @@ jQuery.ajax({
       "id": 66
     }
   ],
+  "latitude": 0.0,
```
Comment (dipth) [inline source/includes/_settings.md]: Please move these attributes above `"makes": [` to keep the list sorted alphabetically

**Comment by andreaseriksson** [comment]
@dipth Updated now

**Pull request merged**